utils/config/daos_server.yml

```diff
@@ -170,11 +170,18 @@
 ## Number of hugepages to allocate for use by NVMe SSDs
 #
 ## Specifies the number (not size) of hugepages to allocate for use by NVMe
-## through SPDK. This indicates the number to be used for each spawned
-## I/O Engines, so the total will be this number * number of I/O Engines.
-## Default system hugepage size will be used.
+## through SPDK. Note that each target requires 1 GiB of hugepage space, so
+## this value needs to represent the total amount of hugespace required for
+## all targets across all engines on host, divided by the system hugepage size.
+## If not set here, an appropriate value will be automatically calculated based
+## on the number of targets (using the default system hugepage size).
+#
```

Comment:
has this auto-calculation been backported to 2.0, or is it only available on master?

Reply:
just on master

Reply:
perhaps we should mention about the extra hugepages that we allocate when we calculate automatically, this could avoid some inconvenience when getting the cryptic DPDK couldn't init failures

Reply:
no-op as you just confirmed that we do not auto-calculate in 2.0...

Reply:
removed the auto-calc sentence...

Reply:
I would prefer "hugepage memory" rather than "hugespace" but that's just a minor nit

Reply:
also I would probably recommend mentioning the need for an additional couple of pages for overhead because otherwise users will encounter DPDK unable to initialise errors